imap_processing/ultra/l1b/de.py

```diff
@@ -1,7 +1,22 @@
-"""Calculate Annotated Direct Events."""
+"""Calculate Direct Events."""
 
 import numpy as np
 
+from imap_processing.cdf.defaults import GlobalConstants
```

Comment:
Is `imap_processing.cdf.defaults` going to go away once all CDF generation is migrated to use `ImapCdfAttributes`?

Reply:
Nice catch. I'll probably model the attributes after what Maxine is doing in her PR. I put it as a TODO for now.